tekton/vendor/git-clone.yaml

```diff
@@ -0,0 +1,144 @@
+# https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.2
```

Comment:
instead of vendoring the files, we could add instructions in the Readme (or a script) to run kubectl commands `kubectl apply -f  https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.2 (or SHA)` 

That should make it less likely to accidentally override the tasks already applied to the dogfooding cluster.

Reply:
Good idea! Done.